test/functional/importer_controller_test.rb

```diff
@@ -0,0 +1,130 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+class ImporterControllerTest < ActionController::TestCase
+  def setup
+    @project = Project.create! :name => 'foo'
+    @tracker = @project.trackers.create(:name => 'Defect')
+    @role = Role.create! :name => 'ADMIN', :permissions => [:import]
+    @user = create_user!(@role, @project)
+    @iip = create_iip_for_multivalues!(@user, @project)
+    @issue = create_issue!(@project, @user)
+    create_custom_fields!(@issue)
+    create_versions!(@project)
+    User.stubs(:current).returns(@user)
+  end
+
+  test 'should handle multi-value fields' do
+    post :result, build_params
+    assert_response :success
+    @issue.reload
+    assert_issue_has_affected_versions(@issue, ['Admin', '2013-09-25'])
```

Comment:
1. just for consistency, should probably assert that the issue doesn't have multi values set before the post?

2. shouldn't you be testing non-version multi values as well?